includes/Core/Authentication/Authentication.php

```diff
@@ -1353,4 +1353,27 @@ private function filter_features_via_proxy( $feature_enabled, $feature_name ) {
 		return $feature_enabled;
 	}
 
+	/**
+	 * Invalid nonce error handler.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $action    Action name.
+	 */
+	public function invalid_nonce_error( $action ) {
```

Comment:
Let's make this method to be static. In this case, we won't need to instantiate a new instance of the Authentication class in the Reset class.

Reply:
If we make it static, we will need to pass $context param to call `$this->context->admin_url( 'splash' )` later in the function.

Reply:
We can use `Plugin::instance()->context()->admin_url( ... )`. It's better than instantiating a new instance of the `Authentication` class in the `Reset.php` file.

Reply:
https://github.com/google/site-kit-wp/pull/3976/commits/bd06a5b10d75b7060778cd1ce3d2f38e076790c8